src/server/api/conversations.js

```diff
@@ -0,0 +1,79 @@
+import _ from 'lodash'
+import User from '../models/user'
+import CampaignContact from '../models/campaign-contact'
+
+export const schema = `
+  input ConversationFilter {
+    assignmentsFilter: AssignmentsFilter
+    campaignsFilter: CampaignsFilter
+    contactsFilter: ContactsFilter
+  }
+
+  type Conversation {
+    texter: User!
+    contact: CampaignContact!
+    campaign: Campaign!
+    assignment: Assignment
+  }
+  
+  type PaginatedConversations {
+    conversations: [Conversation]!
+    pageInfo: PageInfo
+  }
+`
+
+function coerceQueryFieldsToResolverFields(instance, fieldsToRemove) {
```

Comment:
@schuyler1d the u_first_name and cc_last_name are necessary because the data going into this resolver is not from a single model.  It's from a join of several tables that have columns with the same name.  I'll note this in a comment.

Reply:
👍